econml/tests/test_orf.py

```diff
@@ -20,9 +21,9 @@ class TestOrthoForest(unittest.TestCase):
     def setUpClass(cls):
         np.random.seed(123)
         # DGP constants
-        cls.n = 1000
-        cls.d_w = 30
-        cls.support_size = 5
+        cls.n = 5000
```

Comment:
Why change the dimensions?

Reply:
Just to make the tests more stable. I think with the previous nsamples and d_w some tests were flaky